api/v1alpha1/perses_types.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2023 The Perses Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:validation:Enum=json,yaml
+type FileExtension string
+
+type File struct {
+	Folder    string        `json:"folder" yaml:"folder"`
+	Extension FileExtension `json:"extension" yaml:"extension"`
+}
+
+type Database struct {
+	File *File `json:"file,omitempty" yaml:"file,omitempty"`
+}
+
+type Schemas struct {
+	// +kubebuilder:default:="/etc/perses/schemas/panels"
+	PanelsPath string `json:"panels_path,omitempty" yaml:"panels_path,omitempty"`
```

Comment:
I'd suggest something different.
```suggestion
    +kubebuilder:validation:optional
	PanelsPath *string `json:"panels_path,omitempty" yaml:"panels_path,omitempty"`
```

If PanelsPath is nil we should not provide to Perses and than we left Perses take decision what's the default value for those paths.

If this change tomorrow on Perses the Operator will not be impacted by this.